habitat/plan.sh

```diff
@@ -64,40 +58,46 @@ do_prepare() {
   build_line "Ruby ABI version appears to be ${RUBY_ABI_VERSION}"
 
   build_line "Setting link for /usr/bin/env to 'coreutils'"
-  [[ ! -f /usr/bin/env ]] && ln -s $(pkg_path_for coreutils)/bin/env /usr/bin/env
-
-  return 0
```

Comment:
if we always return `0` at the end of this function, we would be saying that this function always succeeds.